apps/web/components/AdditionalCalendarSelector.tsx

```diff
@@ -19,7 +21,7 @@ const ImageOption = (optionProps: OptionProps<{ [key: string]: string; type: App
   return (
     <InstallAppButton
       type={data.type}
-      render={(installProps) => {
+      render={(installProps: RenderPropsTypeGeneric<ButtonBaseProps>) => {
```

Comment:
Any reason these [cannot be inferred](https://handbook.cal.com/engineering/best-practices/prefer-inferred-types-over-explicit-ones)?  I'm looking at [this implementation](https://github.com/calcom/cal.com/blob/43cb9611906da888baf629594d43c603d4c5185a/packages/app-store/components.tsx#L17) and it feels incomplete. The union type is making the types to be ambiguous, hence we get the any here.